packages/drivers/tinylicious-driver/README.md

```diff
@@ -2,6 +2,10 @@
 
 Driver for the `tinylicious` Fluid service.
 
+## Additional Test Exports
+
+Some additional test utilities are provided via `@fluidframework/tinylicious-driver/test-utils`. These are not meant for production use and may change in any minor update.
```

Comment:
Nit: do we need to specify "in any minor update"? Or can we just say something like "may change without notice".

Reply:
I think we want to at least guarantee the no patch breaks for these. That is why I chose minor.
BTW corresponding eng.ms page: https://eng.ms/docs/experiences-devices/opg/office-shared/fluid-framework/fluid-framework-internal/fluid-framework/docs/dev/resources/api-changes